release_notes/364-release-notes.adoc

```diff
@@ -72,5 +72,5 @@ Also includes `roxctl` for use in continuous integration (CI) systems.
 
 | Collector
 | Collects runtime activity in Kubernetes or {ocp} clusters.
-| registry.redhat.io/rh-acs/collector:3.3.2-latest
+| registry.redhat.io/rh-acs/collector:3.2.0-latest
```

Comment:
> it actually should be 3.3.1-latest. 
From the latest comments at https://issues.redhat.com/browse/OSDOCS-2857?focusedCommentId=19431238&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-19431238

@tmalove Can you please update this again.

Reply:
@gaurav-nelson  I will get this done now. Thanks.